external/fv3fit/fv3fit/keras/_models/shared/__init__.py

```diff
@@ -10,8 +10,7 @@
 from .training_loop import TrainingLoopConfig, EpochResult
 from .callbacks import TrainingLoopLossHistory
 from .loss import LossConfig
-from .utils import get_input_vector, standard_denormalize, standard_normalize
-from .sequences import XyArraySequence, XyMultiArraySequence, ThreadedSequencePreLoader
```

Comment:
IIRC `XyMultiArraySequence` used to handle appending halos for the convolutional model, was this moved somewhere else in a previous PR?

Reply:
Ok, I see a `append_halos_tensor` in the convolutional tests so I assume this is where that is done now?

Reply:
Yes, it's done in the get_Xy helper equivalent for the convolutional training function.